package.json

```diff
@@ -32,11 +32,11 @@
     "test:once": "karma start",
     "test:once:ci": "karma start",
     "test": "karma start",
-    "ci": "npm run e2e && npm run test:once:ci",
+    "ci": "npm run e2e:jit && npm run test:once:ci",
     "ci:testall": "npm run e2e:testall && npm run test:once:ci",
     "tslint": "tslint",
     "lint": "npm run tslint \"src/app/**/*.ts\" ",
-    "clean": "npm cache clean && npm run rimraf -- node_modules doc typings coverage dist .awcache dll && npm run clean:compile",
+    "clean": "npm cache clean && npm run clean:compile && npm run rimraf -- node_modules doc typings coverage dist .awcache dll",
```

Comment:
This was in the incorrect order because it deleted the rimraf command before it could complete